### PR TITLE
[CICD-24] Use hosted dockerfile image

### PR DESCRIPTION
**Pull request by sarahricker**
# JIRA Ticket

[CICD-24](https://wpengine.atlassian.net/browse/CICD-24)

## What Are We Doing Here

Building and hosting image on WPE docker hub, then calling the hosted image instead of building it every time. 

**Original: Avg 11-14 seconds**
(Build Docker Container on the fly)
cicd-24-docker-container-build-v3-1-1

**New: 4 seconds :tada:** 
(Build Docker Container from Hosted Docker Image)
cicd-24-docker-container-build-cicd-24-branch

**NewNew: 1 second 🐎**
(Pull Docker Image directly from within action.yml)
cicd-24-load-docker-image-directly

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1 @@
...
-LABEL "com.github.actions.description"="An action to deploy your repository to WP Engine via the SSH Gateway"
-LABEL "com.github.actions.icon"="upload-cloud"
-LABEL "com.github.actions.color"="blue"
-LABEL "repository"="http://github.com/wpengine/github-action-wpe-site-deploy"
-LABEL "maintainer"="Alex Zuniga <alex.zuniga@wpengine.com>"
-RUN apk add bash php
-ADD entrypoint.sh /entrypoint.sh
-ADD exclude.txt /exclude.txt
-ENTRYPOINT ["/entrypoint.sh"]
+FROM wpengine/gha:v1
```
Comment (daniel-savo) [inline Dockerfile:1]: Is this still necessary? Can we not simply point to the newly uploaded image in DockerHub in our action.yml?

**Review by daniel-savo — COMMENTED** [review]
Just had one question before approving.

File: Dockerfile (hunk 1 above)
Comment (sarahricker) [reply Dockerfile:1]: Ooo maybe. I'll try it next sprint.

Comment (sarahricker) [reply Dockerfile]: Switched that up in [d364851](https://github.com/wpengine/github-action-wpe-site-deploy/pull/36/commits/d364851dd1790471d944cb0d5e1b060227f4ae74) --- I think that shaved off another 3 seconds!
cicd-24-load-docker-image-directly

**Review by daniel-savo — APPROVED** [review]
LGTM

**Pull request closed without merging**